### ISSUE TITLE
Link failure of validation set

### ISSUE THREAD

**Issue by coderhss** [chenjun2hao/CenterFace.pytorch#10]
Hi：
    Could you please share the link again?

**Comment by chenjun2hao** [comment]
you can download from this project.there is some thing wrong with the share date

**Comment by Gregory1994** [comment]
@chenjun2hao hi, i'd like to know from which project i can download the validation set?
